demisto_sdk/commands/common/tools.py

```diff
@@ -605,8 +605,8 @@ def find_type(path: str):
                 return 'layout'
             else:
                 return 'dashboard'
-
-        elif 'id' in _dict:
+        # When using it for all files validation- sometimes 'id' can be integer
+        elif 'id' in _dict and isinstance(_dict.get('id'), str):
```

Comment:
what will happen in case it is an integer? what will the validation print?

Reply:
I till raise error because in the next line we do `id.lower()`

Reply:
im ok with an error, just asking what it looks like? is it indicative?

Reply:
![image](https://user-images.githubusercontent.com/41257953/79759766-c62ca800-8327-11ea-9d64-5273c33b471d.png)

Reply:
It's not indicative and it's unexpected so it breaks the validation process
![image](https://user-images.githubusercontent.com/41257953/79760065-2de2f300-8328-11ea-8755-a5053d51c849.png)

Reply:
ok lets handle that in the `else` clause of that `elif`